website/pages/docs/drivers/external/lxc.mdx

```diff
@@ -71,7 +71,12 @@ The `lxc` driver supports the following configuration in the job spec:
   }
   ```
 
-- `volumes` - (Optional) A list of `host_path:container_path` strings to bind-mount host paths to container paths. Mounting host paths outside of the allocation directory can be disabled on clients by setting the [`volumes_enabled`](#volumes_enabled) option set to false. This will limit volumes to directories that exist inside the allocation directory.
+- `volumes` - (Optional) A list of `host_path:container_path` strings to
+  bind-mount host paths to container paths. Mounting host paths outside of the
+  [allocation working directory] is allowed by default. You can prevent
+  mounting host paths outside of the [allocation working directory] on
+  individual clients by setting the [`volumes_enabled`](#volumes_enabled)
+  option to `false` in the client's configuration```
```

Comment:
Looks like a rogue fence made it in there from edits
```suggestion
  option to `false` in the client's configuration
```

Reply:
Oops, thanks!